spread.yaml

```diff
@@ -472,9 +473,20 @@ repack: |
     elif ! git show-ref "$DELTA_REF" > /dev/null; then
         cat <&3 >&4
     else
-        trap "rm -f delta-ref.tar current.delta" EXIT
+        tmpdir="$(mktemp -d)"
+        #shellcheck disable=SC2064
+        trap "rm -rf delta-ref.tar current.delta repacked-current.tar $tmpdir" EXIT
+        if [ "$REPACK_KEEP_VENDOR" = "n" ]; then
```

Comment:
Is there a case where we would keep vendor? If we have no use-case, maybe we can drop the if and just always drop vendor?

Reply:
I can imagine a scenario where you have some local modification to the vendored libraries and want to run a spread with it.